roles/remove_stale_lun/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+## https://github.com/ansible/ansible/issues/22397
+## Ansible 2.3 generates a WARNING when using {{ }} in defaults variables of role
+## this workarounds it until Ansible resolves the issue:
+- name: Initialize variables
+  set_fact:
+    data_center: "{{ data_center | mandatory }}" 
```

Comment:
CI fails because of this whitespace + missing changelog

Reply:
done